sdk/sqlmanagement/Azure.ResourceManager.Sql/README.md

```diff
@@ -0,0 +1,77 @@
+# Azure SQL Management client library for .NET
+
+This package follows the [new Azure SDK guidelines](https://azure.github.io/azure-sdk/general_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
+
+## Getting started 
+
+### Install the package
+
+Install the Azure SQL management library for .NET with [NuGet](https://www.nuget.org/):
+
+```PowerShell
+Install-Package Azure.ResourceManager.Sql -Version 1.0.0-preview.1 
```

Comment:
Which version of Install-Package is this?  The version I have installed usign MinimumVersion / MaximumVersion / RequiredVersion, but not Version parameter

Reply:
It's actually Package Manager command, not powershell. Updated.

Reply:
AB#4094